pandapower/pypower/newtonpf.py

```diff
@@ -266,10 +297,11 @@ def newtonpf(Ybus, Sbus, V0, ref, pv, pq, ppci, options, makeYbus=None):
         if tdpf:
             # update the R, g, b for the tdpf_lines, and the Y-matrices
             branch[tdpf_lines, BR_R] = r = r_ref_pu * (1 + alpha_pu * (T - t_ref_pu))
-            Ybus, Yf, Yt = makeYbus(baseMVA, bus, branch)
+            Ybus, Yf, Yt = makeYbus(baseMVA, bus, branch)  # todo expansion with SSC
```

Comment:
this todo should have been addressed no, right?

Reply:
clarified the TODO message. We will address it after completing the FACTS implementations.